src/status_im2/subs/shell.cljs

```diff
@@ -147,3 +147,35 @@
    (let [community-id (:community-id channel)
          community    (get communities (:community-id channel))]
      (community-channel-card community community-id channel channel-id))))
+
+(re-frame/reg-sub
+ :shell/bottom-tabs-notifications-data
+ :<- [:chats/chats]
+ (fn [chats]
+   (let [chats-seq                      (map second chats)
+         unviewed-messages-count-seq    (map #(hash-map (:chat-type %)
+                                                        (or (:unviewed-messages-count %) 0))
+                                             chats-seq)
+         unviewed-mentions-count-seq    (map #(hash-map (:chat-type %)
+                                                        (or (:unviewed-mentions-count %) 0))
+                                             chats-seq)
+         unviewed-messages-count-total  (apply merge-with + unviewed-messages-count-seq)
+         unviewed-mentions-count-total  (apply merge-with + unviewed-mentions-count-seq)
```

Comment:
TIL

Reply:
omg it's so satisfying when you find such solutions :)

Reply:
but yeah sure performance-wise one `reduce` always win :)

Reply:
this looked cool though :)